misc/acrn-config/scenario_config/vm_configurations_c.py

```diff
@@ -355,6 +355,18 @@ def gen_pre_launch_vm(vm_type, vm_i, scenario_items, config):
         print("\t\t\t.size = VM0_TPM_BUFFER_SIZE,", file=config)
         print("\t\t},", file=config)
         print("#endif", file=config)
+
+    if (vm_i == 0 and vm_info.mmio_resource_info.p2sb.get(vm_i) is not None
+        and vm_info.mmio_resource_info.p2sb[vm_i]):
+        print("#ifdef P2SB_BAR_ADDR", file=config)
+        print("\t\t.pt_p2sb_bar = true,", file=config)
+        print("\t\t.mmiodevs[0] = {", file=config)
+        print("\t\t\t.base_gpa = P2SB_BAR_ADDR,", file=config)
```

Comment:
for base_gpa, it should be a calculated address, so better not use MACRO here. please refer implementation of TPM (but do not use prefix of VM0_xxx, which should be fixed when general mmio passthru framework is ready).